api/api-npm/__test__/data/transitive-deps/project/main.aqua

```diff
@@ -0,0 +1,12 @@
+aqua Main
+
+export versionBC, versionAC
+
+use "A.aqua"
```

Comment:
shouldn't it be "A/A.aqua" or something? like we always prefix path with the package name right?